example/hooks/useJoinRoom.ts

```diff
@@ -1,6 +1,6 @@
 import {
   useCamera,
-  useFishjamClient,
+  joinRoom as fjJoinRoom,
```

Comment:
I still don't like this name 😞 But I guess we have to leave it as it is for now, and maybe update example logic in future to avoid such case.
Maybe we can move `useJoinRoom` hook from example to library? 🤔